### DISCUSSION TITLE
How to watch() only for special subfields in array of objects?

### DISCUSSION THREAD

**Discussion by sad8c** [react-hook-form/react-hook-form#1937] [Q&A]
For example:
I have contacts Array Field (`useFieldArray()`). Each contact is an object with contactType and contactValue fields.
I need to watch only for each contactType field in contacts array, because `watch("contacts")` is excessively.
I need something like `watch("contacts[].contactType")`, but it doesn't work.

Thanks!

**Comment by bluebill1049** [comment]
`watch("contacts[].contactType")` don't think we support such look up.

**Reply by sad8c** [reply]
Of course! But maybe some another way?

**Reply by bluebill1049** [reply]
unless you can specify which field you want to watch within the array.
